Assignment2/Assignment2_QNo2/SolAssignmentNo2_QNo2.rb

```diff
@@ -0,0 +1,45 @@
+# => 2. Write a class (with constructor) to define instance
+# => methods for getting area & perimeter of a rectangle.
+
+# => Buisness logic class
+class Rectangle
+  def initialize(length, breadth)
+    @length = length
+    @breadth = breadth
+  end
+  # => return area of a rectangle
+  def get_area
+    @length * @breadth
+  end
+  # => return perimeter of a rectangle
+  def get_perimeter
+    2 * (@length + @breadth)
+  end
+end
+
+# => Define user interface class
+class CallRectangle
+  @@cnt = 1
```

Comment:
@HrisikeshSinghaChowdhury Please use proper class name. Why do we need class variable here?